SU2_CFD/src/iteration_structure.cpp

```diff
@@ -2421,7 +2421,7 @@ void CDiscAdjFluidIteration::RegisterInput(CSolver *****solver_container, CGeome
 
     solver_container[iZone][iInst][MESH_0][ADJFLOW_SOL]->RegisterSolution(geometry_container[iZone][iInst][MESH_0], config_container[iZone]);
 
-    if (turbulent){
+    if ((Kind_Solver == DISC_ADJ_RANS) && !frozen_visc){
```

Comment:
In driver_structure.cpp you replaced all the Kind_Solver checks by a turbulent boolean, why is the opposite case here?

Reply:
I noticed some places where I had forgotten frozen_visc, I'll sift through it to make sure all is consistent, thanks for spotting it.